docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -56,6 +48,44 @@ Python style
           """
           pass
 
+Imports
+-------
+
+* Use convenience imports whenever available. For example, do this::
+
+      from django.views.generic import View
+
+  Don't do this::
+
+      from django.views.generic.base import View
+
+* Put imports in five groups: future, standard library, third-party libraries,
+  other Django components, the same component; sort lines in each group
+  alphabetically by the full module name. Place ``import module`` statements
+  after ``from module import objects``.
+
+* On each line, list imported modules first, then classes, and then other
+  objects also ordering each group alphabetically, with upper case before
+  lower.
+
+* Break long lines using parentheses, indenting continuation lines by 4 spaces.
+  For example::
+
+        from __future__ import unicode_literals
```

Comment:
It would be good to make the example more general(e.g. move it to a new sub-section "Example").

```rst
Imports
-------

* Foo
* Bar

Example
^^^^^^^

::

    from __future__ import unicode_literals

    import email.message

    # ...
```